### ISSUE TITLE
Backend Manual Order - Cannot Search/Select Course Item

### ISSUE THREAD

**Issue by JamesSimpson** [LearnPress/learnpress#442]
So after the update 3 days ago, when manually trying to process an order, you can select the customer it is linked to, but when searching for the course in the popup modal, nothing is displayed.

Running the latest version of LearnPress, on Wordpress 5.4.2

Steps to recreate issue:

1. Learnpress > Orders > Add New
2. Order Items > Add Item > (type start of course name)

Nothing displays at all

**Comment by Marinski** [comment]
Hi, I had the same issue so I wrote in the support forum: https://thimpress.com/forums/topic/add-order-manually-not-working/. They solved my issue on my sandbox site very quickly but I'm still waiting for support to tell me what have they done, so I can replicate the fix on my production site.
While I was waiting, I decided to fix the things by my self in an old-fashioned way. This is what I've done:
1) accessed my server via cPanel File Manager
2) zipped the learnpress folder located on my sandbox site at wp-content/plugins and downloaded the zip
3) deleted the learnpress folder located on my production site at wp-content/plugins and uploaded the zip from point 2
4) extracted the zip and everything works fine now

Here's the zip file I have downloaded, feel free to use it at your own risk and BACKUP your site and database before you do anything (once there is a real fix from LearnPress I will delete the file): https://1drv.ms/u/s!Aphhl88fqkMsvTzQKz1Qux9xAoqR?e=tFK60w

This is not a solution, it's just a workaround until the guys at LearnPress fix the issue.

Regards,
Marin
